examples/tests/AuctionSpec.hs

```diff
@@ -68,38 +68,39 @@ hammerToWithdraw = do
 
 noBids :: MonadBlockChain m => m ()
 noBids = do
-  t0 <- currentTime
+  (_, t0) <- currentTime
   let deadline = t0 + 60_000
+  deadlineNextSlot <- getEnclosingSlot deadline <&> (+ 1)
   offerOref <- A.txOffer (wallet 1) (banana 2) 30_000_000
   A.txSetDeadline (wallet 1) offerOref deadline
-  awaitTime (deadline + 1)
+  awaitSlot deadlineNextSlot
   A.txHammer (wallet 1) offerOref
   return ()
 
 oneBid :: MonadBlockChain m => m ()
 oneBid = do
-  t0 <- currentTime
+  (_, t0) <- currentTime
   let deadline = t0 + 60_000
   offerOref <- A.txOffer (wallet 1) (banana 2) 30_000_000
   A.txSetDeadline (wallet 1) offerOref deadline
   A.txBid (wallet 2) offerOref 30_000_000
-  awaitTime (deadline + 1)
+  awaitEnclosingSlot (deadline + 1)
```

Comment:
Why not wait just until the deadline, as the `hammerValidator` is supposed to allow?

Reply:
That makes sense to me.